RealmTasks Apple/RealmTasks iOS/TablePresenter.swift

```diff
@@ -115,9 +115,9 @@ UITableViewDataSource, UITableViewDelegate, UIGestureRecognizerDelegate where Pa
     // MARK: UITableViewDelegate
 
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        if cellPresenter.currentlyEditingIndexPath?.row == indexPath.row {
-            return floor(cellPresenter.cellHeight(forText: cellPresenter.currentlyEditingCell!.textView.text))
-        }
+//        if cellPresenter.currentlyEditingIndexPath?.row == indexPath.row {
```

Comment:
I thought I had. I just tested it again, and it seems to be wrapping lines properly, even when those lines are commented out.